frontend/src/components/Utils/time.js

```diff
@@ -40,10 +44,12 @@ export function timestampToDateFormatted(msTimestamp, timezone='America/New_York
  * @param {string} timezone Timezone in which to convert.
  * @returns {string} Formatted time.
  */
-export function timestampToFormatted(msTimestamp, timezone = 'America/New_York') {
+export function timestampToFormatted(msTimestamp, timezone = '') {
```

Comment:
Do you need to add the default value to the JSDoc for `timezone` param?

Reply:
oooh yeah I forgot about that! Thanks!